tree/dataframe/inc/ROOT/RSnapshotOptions.hxx

```diff
@@ -20,7 +20,7 @@ namespace ROOT {
 namespace RDF {
 /// A collection of options to steer the creation of the dataset on file
 struct RSnapshotOptions {
-   using ECAlgo = ::ROOT::ECompressionAlgorithm;
+   using ECAlgo = ROOT::ECompressionAlgorithm;
```

Comment:
I think we needed the leading `::` to help visual studio at some point, but if it compiles fine on windows then I guess it's ok to remove them @amadio ?